examples/amp-access-fewcents.html

```diff
@@ -124,6 +121,7 @@
       }
     </style>
     <script async custom-element="amp-access" src="https://cdn.ampproject.org/v0/amp-access-0.1.js"></script>
+    <script async custom-element="amp-access-fewcents" src="https://cdn.ampproject.org/v0/amp-access-fewcents-0.1.js"></script>
```

Comment:
@honeybadgerdontcare can you tell why this would cause the validation checks to throw:

```
examples/amp-access-fewcents.html:124:4 Custom JavaScript is not allowed. (see https://amp.dev/documentation/guides-and-tutorials/learn/validation-workflow/validation_errors/#custom-javascript-is-not-allowed)
```

The validation config included in this PR seems like it should allow this, right?